Tests/Common/Data/SubscriptionManagerTests.cs

```diff
@@ -265,6 +265,147 @@ public void GetsCustomSubscriptionDataTypes()
             Assert.AreEqual(expected, SubscriptionManager.IsSubscriptionValidForConsolidator(subscription, consolidator));
         }
 
+        [TestCase(SecurityType.Future, Resolution.Tick, typeof(Tick), TickType.Trade, null, typeof(TradeBar), true)]
```

Comment:
Since resolution and dataType are always `Resolution.Tick` and `typeof(Tick)`, they could be removed from the test parameters